src/socket-activate/socket-activate.c

```diff
@@ -97,6 +97,17 @@ static int open_sockets(int *ret_epoll_fd) {
                 count++;
         }
 
+        if (count > 1 && !arg_accept && arg_inetd)
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                       "--inetd only supported with a single file descriptor, "
+                                       "or with --accept.");
```

Comment:
Please do not break the error message into multiple lines.
People may grep the code with the whole error message.